### DISCUSSION TITLE
Lay out presentation module

### DISCUSSION THREAD

**Discussion by johannadm** [netwerk-digitaal-erfgoed/solid-crs#377] [Presentation]
For the presentation of the application towards visiting users, we would suggest a lay out that's close to the lay out for administrators. What do you all think of the following mock-ups? We discussed them already a bit with David, but would like to be sure everyone is okay with the prototype. 

1. An extra overview of all the collections.
![afbeelding](https://user-images.githubusercontent.com/80322390/126641056-80178087-2e8a-4521-b6a8-18b15972db0b.png)

2. View of one collection. The rdf file can be downloaded bij clicking on the three dots in the upper right corner.
![afbeelding](https://user-images.githubusercontent.com/80322390/126641201-5ac0e9c6-9097-4362-a15c-573b6be6c571.png)

3. Search results.
![afbeelding](https://user-images.githubusercontent.com/80322390/126641314-29a5327f-06b5-4b3c-a5e1-0beecc504b18.png)

4. Object detail page. The picture can be enlarged and downloaded. If the object was found through searching, there's a link to go back to the search results. If the object was found as part of a collection, the right collection will be highlighted in the menu. The rdf file of an object can be downloaded as well.
![afbeelding](https://user-images.githubusercontent.com/80322390/126641419-e8916c1a-1d5f-4871-a220-b40566d0ed87.png)

5. Enlarged picture. Depends on the size of the original picture.
![afbeelding](https://user-images.githubusercontent.com/80322390/126642047-bd9bfed7-4e8f-41dd-ab71-b2ac99f8cb9a.png)

6. About page with more info and details about the heritage institution.
See [discussion 375](https://github.com/netwerk-digitaal-erfgoed/solid-crs/discussions/375) to discuss the about page and the link to it.

**Comment by ddeboer** [comment]
Some points:

- Screen 1: can you also show the collection’s [`schema:description`](https://github.com/netwerk-digitaal-erfgoed/solid-crs/discussions/162) on the collections list page? (That field is not editable in the admin interface but we can edit it manually in the pod for now.)
- Screen 4: please _always_ highlight the collection in the primary navigation, also when accessing the detail page from search results.
